actions/project.go

```diff
@@ -69,7 +86,56 @@ func DownloadTemplate(c *cli.Context) {
 	utils.DeleteTempFile(zipFileName)
 }
 
-//ValidateProject type
-func ValidateProject() {
-	//code here
+// ValidateProject returns the language and buildType for a project at given filesystem path
+func ValidateProject(c *cli.Context) {
+	projectPath := c.Args().Get(0)
+	language, buildType := determineProjectInfo(projectPath)
+
+	resp := ValidationResponse{
+		Status: "success",
+		Result: ProjectType{language, buildType},
+		Path:   projectPath,
+	}
+
+	projectInfo, err := json.Marshal(resp)
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+	fmt.Println(string(projectInfo))
+}
+
+func determineProjectInfo(projectPath string) (string, string) {
```

Comment:
Too keep this in line with other functions and the possibility that this could be called from outside of this local scope in future, could this be global? i.e `DetermineProjectInfo`?

Reply:
Yeah sure. As I would with node, I only exported what had to be exported. Wouldn't you not make it global only if it became necessary to?

Reply:
Keeping it local until we need to expose it further would probably be best.